distribution/docs/src/main/resources/content/_securityFramework/security-token-service.adoc

```diff
@@ -3,7 +3,7 @@
 :status: published
 :parent: Security Framework
 :children: Security PEP Interceptor
-:order: 08
+:order: 09
```

Comment:
Why the order change here? Is Federated Identity supposed to become order #8?

Reply:
Both the Expansion Service and STS sections had order 08. I thought that was weird.

Reply:
@ricklarsen ?

Reply:
They should be different, so this change is fine. If they are the same, they'll just get ordered alphabetically by file-name, so this seems more "on-purpose"